source/GameInterface/Services/Villages/Patches/VillagePatches.cs

```diff
@@ -13,16 +19,35 @@ internal class VillagePatches
     [HarmonyPrefix]
     private static bool DailyTickPrefix()
     {
+        if(ModInformation.IsServer) return true;
         return false;
     }
```

Comment:
This does the same thing

```suggestion
    private static bool DailyTickPrefix() => ModInformation.IsServer;
```